test/conftest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bats
+
+# LABELS for test_images - positive cases
+@test "/project/image/policy/deprecated-labels" {
+  run conftest test --policy $POLICY_PATH/image/deprecated-labels.rego label.json
```

Comment:
The command is missing the `--namespace required_checks` which was introduced in #9

Reply:
This applies to all subsequent `conftest test` commands in this file.